### PR TITLE
Div mod simplify in index expressions

### PR DESCRIPTION
**Pull request by SamirDroubi**

### CODE REVIEW

**Comment by SamirDroubi** [comment]
Fixes #312

File: src/exo/LoopIR_scheduling.py [hunk 1]
```diff
@@ -3139,6 +3287,20 @@ def check_quot(const, div):
 
         return check_quot(quot.lhs, quot.rhs) or check_quot(quot.rhs, quot.lhs)
 
+    @staticmethod
+    def simplify_div(e):
```
Comment (yamaguchi1024) [inline src/exo/LoopIR_scheduling.py:3291]: What is this? Do you use it?

File: src/exo/range_analysis.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+from .LoopIR import LoopIR
+
+
+class AffineIndexRangeAnalysis:
```
Comment (yamaguchi1024) [inline src/exo/range_analysis.py:4]: This class needs a comment here in explaining what it does.

File: src/exo/LoopIR_scheduling.py (hunk 1 above)
Comment (SamirDroubi) [reply src/exo/LoopIR_scheduling.py]: Done

File: src/exo/range_analysis.py (hunk 2 above)
Comment (SamirDroubi) [reply src/exo/range_analysis.py]: Done

**Review by yamaguchi1024 — APPROVED** [review]

**Pull request merged**